build/cmake/CMakeLists.txt

```diff
@@ -7,9 +7,12 @@
 # in the COPYING file in the root directory of this source tree).
 # ################################################################
 
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 3.0.0...3.13.2 FATAL_ERROR)
+
+project(zstd
+VERSION 1.3.7
```

Comment:
Is there a way to determine version programmatically from `lib/zstd.h` ?
It would avoid future synchronization errors.